queue_and_stack/dll_queue.py

```diff
@@ -1,19 +1,43 @@
 import sys
-sys.path.append('../doubly_linked_list')
+
+sys.path.append("../doubly_linked_list")
 from doubly_linked_list import DoublyLinkedList
 
 
 class Queue:
-    def __init__(self):
+    def __init__(self, node=None):
         self.size = 0
         # Why is our DLL a good choice to store our elements?
-        # self.storage = ?
+        self.storage = DoublyLinkedList(self.size)
```

Comment:
I'm also interested as to why this still works with "self.size" passed in as an argument here!